src/localization/en.js

```diff
@@ -16,9 +16,17 @@ export const en = {
     passwordHint: 'Enter your password',
     invalidCredentials: 'Invalid credentials',
   },
+  sessionId: {
+    invalidCredentials: 'No se genero el sessionId',
```

Comment:
Spotted some words in Spanish 😉 

ℹ️ Suggestion for the future: if you want to localize your app, create a new file called `es.js` and create your language file in Spanish. `react-native-localization` can use your device's language by default if configured properly.

Reply:
Perfect!